src/hotspot/share/gc/serial/serialHeap.hpp

```diff
@@ -41,6 +41,8 @@ class SerialHeap : public GenCollectedHeap {
   MemoryPool* _survivor_pool;
   MemoryPool* _old_pool;
 
+  size_t _live_size;
```

Comment:
Please rename to `_live_estimate` like the others. Avoid having different names in different collectors for the same thing.

Reply:
This field is unused. Removed.